config/operator/crd/bases/scylla.scylladb.com_scyllaclusters.yaml

```diff
@@ -1438,6 +1438,15 @@ spec:
                   description: PollInterval specifies how often upgrade logic polls on state updates. Increasing this value should lower number of requests sent to apiserver, but it may affect overall time spent during upgrade.
                   type: string
               type: object
+            multiDcCluster:
```

Comment:
just thinking out loud, instead of talking of multiDcCluster, I'm wondering if remoteCluster or externalCluster wouldn't be more appropriate as "dc" may be a bit too restrictive in term of deployment strategies.

Reply:
I've initially named it remote cluster but then decided to respect the naming of the feature in scylla: https://docs.scylladb.com/operating-scylla/procedures/cluster-management/create_cluster_multidc/ which I still think is better

Reply:
make sense. Thanks for the reference.